qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -141,6 +167,20 @@ class FermionicOp(SecondQuantizedOp):
 
     are possible labels.
 
+    As a programmatic approach, you can input a list of tuples as the labels,
+    too. Thus, a FermionicOp can be initialized using a list of tuples of the
+    form, `(action, index)`. `action` is an integer where `0` denotes an
+    annihilation and `1` a creation operation. The `index` is also an integer
```

Comment:
I have started to wonder whether it's really that beneficial to use an integer for the `action` here.
Personally, I would find `"+"` and `"-"` a little bit less confusing and it surely is not too much of a hassle to write.

What do you think?

Reply:
That's a good question actually. I don't see why we should not support those. Especially `I`. Because `N` and `E` can be expressed in terms of `+` and `-`, but in the current implementation we cannot construct an identity operation with a certain coefficient via this interface, right?

Reply:
The identity operation should be constructed by proving an empty tuple:
```python
FermionicOp(((), 2.0))
```

Reply:
OK. I don't support N and E.
I'll support `FermionicOp(([()], 2.0))`. 

This is for programming, so I want to only support integer.

Reply:
@ikkoham The syntax should be `FermionicOp(([], 2.0))`, not `FermionicOp(([()], 2.0))`. You can see this because the string syntax is `FermionicOp(("", 2.0))` and `""` corresponds to `[]`, not `[()]`. There is no need to handle an empty tuple. In my comment above, I should have used an empty list instead of an empty tuple to be more clear.

Reply:
@kevinsung right. empty list is correct.